package.json

```
@@ -1,199 +1,90 @@
-    "lint:eslint": "eslint '**/*.{js,jsx}'",
-    "lint:types": "yarn tsc",
-    "lint-fix": "yarn lint:eslint --fix",
-    "postinstall": "patch-package",
-    "test": "CI=true react-app-rewired test",
-    "test:watch": "react-app-rewired test",
-    "test:xs": "exit 0",
-    "test:debug": "react-app-rewired --inspect-brk test --runInBand --no-cache",
-    "eject": "react-scripts eject"
-  },
-  "prettier": {
-    "arrowParens": "avoid",
-    "trailingComma": "all",
-    "singleQuote": true
-  },
-  "jest": {
-    "transformIgnorePatterns": [
-      "/node_modules/(?!@lit-labs)/"
-    ]
+    "eslint-plugin-prettier": "^3.2.0",
+    "prettier": "^1.18.2",
+    "rollup-plugin-minify-html-literals": "^1.2.6",
+    "rollup-plugin-multi-input": "^1.3.1",
+    "rollup-plugin-summary": "^1.3.0",
+    "rollup-plugin-terser": "^7.0.2",
+    "ses": "^0.14.1",
+    "typescript": "^4.2.3"
   },
   "eslintConfig": {
-    "ignorePatterns": [
-      "**/*.umd.js",
-      "**/generated/*"
+    "extends": [
+      "@agoric"
     ],
-    "parser": "babel-eslint",
-    "parserOptions": {
-      "ecmaFeatures": {
-        "jsx": true
-      },
-      "ecmaVersion": 2018,
-      "sourceType": "module"
-    },
-    "plugins": [
-      "jest"
+    "ignorePatterns": [
+      "build/**"
     ],
     "rules": {
-      "import/no-extraneous-dependencies": [
-        "error",
-        {
-          "devDependencies": [
-            "**/*.test.*",
-            "**/setupTests.js"
-          ]
-        }
-      ],
-      "react/prop-types": "off",
-      "react/react-in-jsx-scope": "off",
-      "react/display-name": "off",
-      "@typescript-eslint/no-floating-promises": "off"
-    },
-    "env": {
-      "browser": true,
-      "jest/globals": true
-    },
-    "settings": {
-      "import/resolver": {
-        "node": {
-          "extensions": [
-            ".js",
-            ".jsx"
-          ]
-        }
-      },
-      "react": {
-        "version": "detect"
-      }
-    },
-    "extends": [
-      "@agoric",
-      "plugin:react/recommended"
-    ]
-  },
-  "eslintIgnore": [
-    "dist/",
-    "conf/",
-    "build/",
-    ".cache",
-    "node_modules",
-    "config-overrides/"
-  ],
-  "files": [
-    "build/"
-  ],
-  "staticFiles": {
-    "staticPath": [
-      {
-        "staticPath": "public/lib",
-        "staticOutDir": "lib"
-      },
-      {
-        "staticPath": "public/assets"
-      }
-    ],
-    "excludeGlob": "**/*.js",
-    "watcherGlob": "**/*.html"
+      "import/no-extraneous-dependencies": "off"
+    }
   },
-  "browserslist": {
-    "production": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
+  "prettier": {
+    "trailingComma": "all",
+    "singleQuote": true
   },
   "publishConfig": {
     "access": "public"
+  },
+  "engines": {
+    "node": ">=11.0"
```

this is a private package so let's make maintenance easier on ourselves and use the active LTS.
```suggestion
    "node": ">=18.0"
```